src/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,432 @@
+import { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+import { Placeholder } from '../_localComponents/Placeholder';
+import { Accordion } from './Accordion';
+
+const meta: Meta<typeof Accordion> = {
+  title: 'Components/Accordion',
+  component: Accordion,
+  tags: ['autodocs'],
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Accordion>;
+
+const accordions = [
+  {
+    id: 0,
+    triggerCopy: 'Does Chromatic replace Jest or Enzyme?',
+    icon: 'cloudhollow',
+  },
+  {
+    id: 1,
+    triggerCopy: 'Can I get rid of my BrowserStack or Sauce subscription?',
+    icon: 'facehappy',
+  },
+  {
+    id: 2,
+    triggerCopy: 'How is this compared to Selenium, Cypress, or Playwright?',
+    icon: 'accessibilityalt',
+  },
+  { id: 3, triggerCopy: 'Why run visual tests in the cloud?', icon: 'heart' },
+];
+
+const PanelContent = ({ inverse }) => {
```

Comment:
Not a requirement, but I'm wondering if it is worth adding default typography styling for the panel content. That way we can just drop in the component as is for FAQs. But folks can still override it for other use cases.

Reply:
@winkerVSbecks I'm not 100% clear on what you're suggesting here. Or, rather, I'm not 100% clear on _how or where_ you're suggesting that happens, maybe. The location of this comment versus its content are throwing me, I think. 🙃

Reply:
The contents of `Accordion.Panel` are basically unstyled at the moment. Does it make sense add some default styles (it could match the FAQ content design).

Reply:
Ok, that's what I thought you meant and I think why I was confused. Because `Accordion.Panel` is already being styled [in `Accordion.tsx` here](https://github.com/chromaui/tetra/pull/72/files#diff-f90c99d391b33619f0416a684e0cb572dbd44476f00b04f51f889dfa0ae5a2d6R125-R144). It's defining default styles there and also has an immediate child star selector to address color issues of nested components. What you've highlighted here is just the lorem ipsum block for the stories. It's only taking `inverse` as a prop because `Text` doesn't play nicely with style inheritance. I've changed this to `ExamplePanelContent` to help with confusion.

If I'm still missing the mark, let's hop on a call. 🙂